cog.c

```diff
@@ -352,6 +356,23 @@ on_create_view (CogShell *shell, void *user_data G_GNUC_UNUSED)
 #endif
                                                       NULL);
 
+    if (s_options.background_color != NULL) {
+      gboolean has_valid_color = FALSE;
+#if !COG_USE_WEBKITGTK
+#if WEBKIT_CHECK_VERSION(2, 23, 0)
+      WebKitColor color;
+      has_valid_color = webkit_color_parse (&color, s_options.background_color);
+#endif
+#else
+      GdkRGBA color;
+      has_valid_color = gdk_rgba_parse (&color, s_options.background_color);
+#endif
+
+#if defined(webkit_web_view_set_background_color)
+      if (has_valid_color)
+        webkit_web_view_set_background_color (web_view, &color);
```

Comment:
If the color cannot be parsed, it would be desirable to print an error and exit instead of ignoring the specified value. Could you please add an `else` block here for that?